import-automation/executor/app/executor/cloud_run_simple_import.py

```diff
@@ -0,0 +1,217 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utilities to run simple imports as cloud run jobs.
+
+For mre details on simple imports, please refer to:
+https://github.com/datacommonsorg/import/tree/master/simple
+"""
+import datetime
+import os
+import sys
+import time
+
+from absl import logging
+import pytz
+
+# Set path for import modules.
+_SCRIPT_DIR = os.path.abspath(os.path.dirname(__file__))
+sys.path.append(_SCRIPT_DIR)
+sys.path.append(os.path.join(_SCRIPT_DIR.split('/data/', 1)[0], 'data', 'util'))
+
+import cloud_run
+import file_util
+
+# Default settings.
+_DEFAULT_PROJECT = 'datcom-import-automation-prod'
+_DEFAULT_SIMPLE_IMAGE = 'gcr.io/datcom-ci/datacommons-simple:stable'
+_DEFAULT_LOCATION = 'us-central1'
+_DEFAULT_GCS_BUCKET = 'datcom-prod-imports'
+_DEFAULT_CONFIG_PREFIX = 'import_config'
+
+# Path for simple import config spec under data/scripts/simple
+_IMPORT_CONFIG_DIR = '/simple_imports/'
+
+
+def get_simple_import_job_id(import_spec: dict, import_script: str) -> str:
+    """Returns the job id if the script is a simple import config, else ''.
+
+  Args:
+    import_spec: Import specification dictionary with import name, script.
+    import_script: Specific import script being processed
+
+  Returns:
+    job_is as a string simple-import-<import_name> for simple-imports
+    or '' for other imports.
+  """
+    # Create a job is for simple import cloud run as
+    # <config_path>-<import_name>
+    import_name = import_spec.get('import_name')
+    import_pos = import_script.find(_IMPORT_CONFIG_DIR)
+    if import_pos < 0 or _DEFAULT_CONFIG_PREFIX not in import_script:
+        # Not a simple import
+        return ''
+    # Get the import config path under .../simple_imports/
+    job_str = [
```

Comment:
I think it does but wanted to make sure this accounts for sub-folders.

e.g. say UN has 2 imports - says unsdg and undp - and they are specified in  `simple_imports/un/unsdg/import_config.json` and `simple_imports/un/undp/import_config.json` respectively, will their resulting job names be `un-unsdg` and `un-undp`?

Reply:
yes, in addition to sub directory, the import name can be different:
```python -c "import cloud_run_simple_import as cr; print(cr.get_simple_import_job_id({'import_name': 'UNSDGData'}, '/abc/def/data/simple_imports/un/unsdg/import_config.json'))"
simple-imports-un-unsdg--unsdgdata
```